build-farm/make-adopt-build-farm.sh

```diff
@@ -126,6 +127,7 @@ if [ -z "${JDK_BOOT_VERSION}" ]
 then
   echo "Detecting boot jdk for: ${JAVA_TO_BUILD}"
   echo "Found build version: ${JAVA_FEATURE_VERSION}"
+  # shellcheck disable=SC2004
   JDK_BOOT_VERSION=$(($JAVA_FEATURE_VERSION-1))
```

Comment:
Might as well fix this one instead of bypassing the check

```suggestion
  JDK_BOOT_VERSION=$(( JAVA_FEATURE_VERSION - 1 ))
```

Reply:
Fair. I didn't implement this one as I was unsure on when `JAVA_FEATURE_VERSION` is a string and when it is a int